### PR TITLE
Update files outdated Korean docs in dev-1.16-ko.5

### PR DESCRIPTION
**Pull request by novice81**
### 19 files to be modified

- [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x]  content/en/docs/concepts/containers/container-lifecycle-hooks.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/concepts/overview/working-with-objects/names.md 1 file changed, 1 insertion(+), 2 deletions(-)
- [x]  content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 22 insertions(+), 22 deletions(-)
- [x]  content/en/docs/concepts/workloads/controllers/replicationcontroller.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/concepts/workloads/controllers/statefulset.md 1 file changed, 1 insertion(+)
- [x]  content/en/docs/concepts/workloads/pods/init-containers.md 1 file changed, 4 insertions(+), 27 deletions(-)
- [x]  content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x]  content/en/docs/reference/_index.md 1 file changed, 2 deletions(-)
- [x]  content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/tasks/administer-cluster/cluster-management.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 6 insertions(+), 6 deletions(-)
- [x]  content/en/docs/tasks/tools/install-minikube.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/tutorials/hello-minikube.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x]  content/en/docs/tutorials/kubernetes-basics/_index.html 1 file changed, 2 insertions(+)
- [x]  content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html 1 file changed, 1 insertion(+), 1 deletion(-)
- [x]  content/en/docs/tutorials/kubernetes-basics/update/update-intro.html 1 file changed, 2 insertions(+), 2 deletions(-)

from #17488 
/language ko

### CODE REVIEW

**Comment by seokho-son** [comment]
@novice81 Thank you for your contribution :)
I am going to review this PR.
/assign @seokho-son

File: content/ko/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -277,10 +277,11 @@ kubelet에 의해서 재시작되는 종료된 컨테이너는
...
 
-일반적으로, 파드는 누군가 파드를 파괴할 때까지 사라지지 않는다.
-그것은 주로 사람이나 컨트롤러에 의해서 일어난다.
-이 법칙에 대한 유일한 예외는 일정 기간(마스터의 `terminated-pod-gc-threshold`에 의해 결정되는)
-이상 파드의 `phase`가 Succeeded 또는 Failed라서 파드가 만료되고 자동적으로 파괴되는 경우이다.
+일반적으로, 파드는 사람 혹은 컨트롤러의 프로세스가 명시적으로 파드를 삭제할 때까지 남아 있다.
+컨트롤 플레인은 파드의 수가 설정된 임계치 (kube-controller-manager에서
+`terminated-pod-gc-threshold`에 의해 결정되는) 를 초과할 때,
+종료된 파드들 (`Succeeded` 또는 `Failed` 단계) 을 정리한다.
+이로써 시간이 지남에 따라 파드가 생성되고 종료되어 리소스 누수를 피할 수 있다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/workloads/pods/pod-lifecycle.md]: +This avoids a resource leak as Pods are created and terminated over time.

원문의 해석에 다소 차이가 있는 것 같습니다. 참고하시기 바랍니다.
```suggestion
이로써 시간이 지남에 따라 파드들이 생성 및 종료되며 발생하는 리소스 누수를 피할 수 있다.
```

File: content/ko/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -286,8 +286,7 @@ systemctl restart containerd
 
 `systemd` cgroup driver를 사용하려면, `/etc/containerd/config.toml`의 `plugins.cri.systemd_cgroup = true`을 설정한다.
 kubeadm을 사용하는 경우에도 마찬가지로, 수동으로
-[cgroup driver for kubelet](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-master-node)을 
-설정해준다.
+[cgroup driver for kubelet](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node)을 설정해준다.
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/container-runtimes.md]: 기존에도 번역이 누락되어 있었네요. 아래와 같이 바꾸면 어떨까요? 참고하시기 바랍니다.
```suggestion
[kubelet을 위한 cgroup 드라이버](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#configure-cgroup-driver-used-by-kubelet-on-control-plane-node)를 설정한다.
```

File: content/ko/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -281,7 +281,7 @@ DaemonSet 컨트롤러에 의해 생성된 파드는 쿠버네티스 스케줄
 
 쿠버네티스 스케줄러는 노드 상에 모든 노드에 대해 충분한 리소스가 존재하도록 보장한다.
 노드 상에 컨테이너에 대한 요청의 합이 노드 용량보다 더 크지 않도록 체크한다.
-kubelet에 의해 구동된 모든 컨테이너를 포함하지만, [컨테이너 런타임](/ko/docs/concepts/overview/components/#노드-컴포넌트)에 의해 직접 구동된 컨테이너 또는 컨테이너 외부에서 동작하는 임의의 프로세스는 해당되지 않는다.
+kubelet에 의해 구동된 모든 컨테이너를 포함하지만, [컨테이너 런타임](/ko/docs/concepts/overview/components/#컨테이너-런타임)에 의해 직접 구동된 컨테이너 또는 컨테이너 외부에서 동작하는 임의의 프로세스는 해당되지 않는다.
 
 파드 형태가 아닌 프로세스에 대해 명시적으로 리소스를 확보하려면,
 [reserve resources for system daemons](/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved) 튜토리얼을 따른다.
```
[the remark this answers is not in the document]
Comment (novice81) [reply content/ko/docs/concepts/architecture/nodes.md]: 감사합니다! 👍

File: content/ko/docs/concepts/workloads/controllers/statefulset.md [hunk 4]
```diff
@@ -260,6 +260,7 @@ web-0이 실패할 경우 web-1은 web-0이 Running 및 Ready 상태가
 
 * [스테이트풀 애플리케이션의 배포](/ko/docs/tutorials/stateful-application/basic-stateful-set/)의 예시를 따른다.
 * [카산드라와 스테이트풀셋 배포](/ko/docs/tutorials/stateful-application/cassandra/)의 예시를 따른다.
+* [Running a replicated stateful application](/docs/tasks/run-application/run-stateless-application-deployment/)의 예시를 따른다.
```
[the remark this answers is not in the document]
Comment (novice81) [reply content/ko/docs/concepts/workloads/controllers/statefulset.md]: 말씀하신대로 한영 병기하고, 링크의 내용은 master 내용이 변경됐을때 함께 변경해보겠습니다. 감사합니다!

File: content/ko/docs/concepts/workloads/pods/init-containers.md [hunk 5]
```diff
@@ -69,6 +68,9 @@ weight: 40
 * 앱 컨테이너들은 병렬로 실행되는 반면, 초기화 컨테이너들은 어떠한 앱
   컨테이너라도 시작되기 전에 실행 완료되어야 하므로, 초기화 컨테이너는 사전 조건들이
   충족될 때까지 앱 컨테이너가 시동되는 것을 막거나 지연시키는 간편한 방법을 제공한다.
+* 앱 컨테이너 이미지의 보안성을 떨어뜨릴 수도 있는 유틸리티 혹은 커스텀 코드를 안전하게
```
[the remark this answers is not in the document]
Comment (novice81) [reply content/ko/docs/concepts/workloads/pods/init-containers.md]: 위 문장과 연결되는 내용을 보면 위쪽에서는 주어가 가까이 있어서 쉽게 읽히지만 아래쪽에서 이렇게 생략하니 어색하게 보이는 것 같아서, 제가 수정하는 부분만 제안해주신 방향대로 주어를 추가해보려고 합니다.

**Comment by seokho-son** [comment]
@novice81 Thank you !!
/lgtm
/approve

**Pull request merged**